docs/advanced_documentation/c-api.md

```diff
@@ -134,6 +175,16 @@ You do not need to call `PGM_buffer_set_nan` on output buffers,
 because the buffer will be overwritten in the calculation core with the real output data.
 ```
 
+### Attribute buffers
+
+An attribute buffer contains data for a single component attribute and represents one column in a columnar representation of component data.
+A combination of attribute buffers with the same amount of elements has the power to carry the same information as row-based [component buffers](#component-buffers).
+
+The type (implying the size) of each attribute can be found using the `PGM_meta_attribute_ctype`.
+
+Since all attributes consist of primitive types, operations are straightforward.
+We therefore do not provide explicit interface functionality.
```

Comment:
Maybe it is a good idea to briefly mention something about the `*set_attribute_buffer()` functionality, or point towards it. Even thought this is more related with the datasets, this is basically the only way to "interact" with "attribute buffers". What do you think?

Reply:
Can you add that? Specifically below in the section on the dataset

Reply:
Actually, i completely forgot that part so feel free to update there